### PR TITLE
Add kroma support (#11179)

### PR DESCRIPTION
**Pull request by simsonraj**
## Motivation
Enable Wemix & Kroma on CCIP

## Solution
Cherry-pick Wemix & Kroma chain changes from Chainlink Core excluding the integration-tests changes as a lot has been changed since the last version of CTF merge

### CODE REVIEW

File: core/chains/evm/config/toml/defaults/WeMix_Mainnet.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# WeMix emits a block every 1 second, regardless of transactions
+LogPollInterval = '3s'
+NoNewHeadsThreshold = '30s'
+
+[OCR]
+ContractConfirmations = 1
+
+[GasEstimator]
+EIP1559DynamicFees = true
+TipCapDefault = '100 gwei'
```
Comment (RensR) [inline core/chains/evm/config/toml/defaults/WeMix_Mainnet.toml:14]: This feels high?

Comment (mohamed-mehany) [reply core/chains/evm/config/toml/defaults/WeMix_Mainnet.toml:14]: It's the default maxPriorityFeePerGas according to: https://docs.wemix.com/v/en/design/eip1559

Comment (simsonraj) [reply core/chains/evm/config/toml/defaults/WeMix_Mainnet.toml:14]: cc @mohamed-mehany

Comment (simsonraj) [reply core/chains/evm/config/toml/defaults/WeMix_Mainnet.toml:14]: @RensR : its wemix's default value as per this doc (towards the end) https://docs.wemix.com/v/en/design/eip1559

File: integration-tests/contracts/contract_loader.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package contracts
 
 import (
 	"errors"
+
```
Comment (HelloKashif) [inline integration-tests/contracts/contract_loader.go:5]: @simsonraj just wondering, this does not include the `contract_deployer.go` change and the `go.mod` CTF version bump changes.
Are those no needed for ccip?

[Ref 1](https://github.com/smartcontractkit/chainlink/pull/11179/files#diff-57c586bd82760e110898453cffd814c6a84977e17b93e11394e5a8531c365bc8R176-R248)

[Ref 2](https://github.com/smartcontractkit/chainlink/pull/11179/files#diff-7c016cae50e2281d7a89e537d58b33cb9e01501820ff9301365fc006c94b712bR25)

Comment (simsonraj) [reply integration-tests/contracts/contract_loader.go:5]: we have some issues merging the integration-tests changes, due to CTF version changes, it didn't work with cherry picking, so excluded that entirely

**Review by mohamed-mehany — APPROVED** [review]
LGTM!

**Review by HelloKashif — APPROVED** [review]
LGTM

**Review by davidcauchi — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Pull request merged**